### PR TITLE
Database redesign changes

### PR DESCRIPTION
**Pull request by mbauckhage**
Changes in App.py to fit to new database structure.

### CODE REVIEW

File: ebike_city_tools/metrics.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def od_sp(G, od, weight, weight_od_flow=False):
         if row["trips"] == 0:
             continue
         # compute shortest path
-        sp_len = nx.shortest_path_length(G, source=row["s"], target=row["t"], weight=weight)
+        sp_len = nx.shortest_path_length(G, source=row["source"], target=row["target"], weight=weight)
```
Comment (NinaWie) [inline ebike_city_tools/metrics.py]: This is a breaking change for other parts of the backend. In my evaluation scripts, the attributes are probably called "s" and "t" somewhere, so I can't merge this change. A good solution in these cases is usually to add these names as keyword arguments, like 
```
def od_sp(G, od, weight, weight_od_flow=False, source_col_name="s", target_col_name="t"):
       .....
       sp_len = nx.shortest_path_length(G, source=row[source_col_name], target=row[target_col_name], weight=weight)
```

**Review by NinaWie — CHANGES_REQUESTED** [review]
Hi @mbauckhage and @eelliinnaa, I just had a look at your changes, because I would suggest that we merge your changes into my repo, then I adapt the evaluation function, and then you can pull from my repo (with the `git pull upstream main` command). I assume that you pushed all changes and this is the latest version of the forked repo?
I made one comment where it would be great if you can adapt the code in the `metrics.py` file to make it more flexible. If you make this change and push to your fork, it should automatically update the pull request and I can review the new version then. Let me know if you have questions.

File: ebike_city_tools/metrics.py (hunk 1 above)
Comment (eelliinnaa) [reply ebike_city_tools/metrics.py]: I changed it back to the original naming and adjusted it on our side. So no changes needed in the metrics.py file.

**Review by NinaWie — APPROVED** [review]

**Comment by NinaWie** [comment]
Thanks @eelliinnaa for the changes, merging this now! You can sync your version with mine by doing `git pull upstream main` from the command line, or by pressing the "Sync" button in your github repo.

**Pull request merged**